packages/babel-plugin-transform-classes/README.md

```diff
@@ -119,3 +119,37 @@ class Bar extends Foo {
 When `Bar.prototype.foo` is defined it triggers the setter on `Foo`. This is a
 case that is very unlikely to appear in production code however it's something
 to keep in mind.
+
+### `builtins`
+
+`Array<string>`, defaults to `[]`.
+
+When extending a native constructor (e.g., `class extends Array {}`), it needs
+to be wrapped.
+Babel only recognizes built-in JavaScript functions defined in the ECMAScript
```

Comment:
Does this option override or augment the default list? If it augments, I'd consider making that more clear in the option name.